### PR TITLE
Fixing some doc comments

### PR DESCRIPTION
**Pull request by srawlins**

### CODE REVIEW

File: lib/logging.dart [hunk 1]
```diff
@@ -123,11 +123,11 @@ class Logger {
...
-  /**
-   * Returns an stream of messages added to this [Logger]. You can listen for
-   * messages using the standard stream APIs, for instance:
-   *    logger.onRecord.listen((record) { ... });
-   */
+  /// Returns a stream of messages added to this [Logger].
+  ///
+  /// You can listen for messages using the standard stream APIs, for instance:
+  ///
+  ///     logger.onRecord.listen((record) { ... });
```
Comment (kevmoo) [inline lib/logging.dart]: Using

```dart
code here
```

instead of indenting

Comment (kevmoo) [inline lib/logging.dart]: triple-tick dart
code
triple-tick

**Comment by kevmoo** [comment]
Otherwise, LGTM

**Comment by srawlins** [comment]
Done! I don't have a merge button, jsyk.

**Comment by kevmoo** [comment]
Would you flag as +1 in the pubspec and update the changelog?

**Comment by srawlins** [comment]
Done! Ready to bump.

**Comment by kevmoo** [comment]
Committed in c2c7765

**Pull request closed without merging**